code/modules/surgery/bodyparts/dismemberment.dm

```diff
@@ -51,25 +51,25 @@
 /obj/item/bodypart/chest/dismember()
 	if(!owner)
 		return FALSE
-	var/mob/living/carbon/C = owner
+	var/mob/living/carbon/chest_owner = owner
 	if(!dismemberable)
 		return FALSE
-	if(HAS_TRAIT(C, TRAIT_NODISMEMBER))
+	if(HAS_TRAIT(chest_owner, TRAIT_NODISMEMBER))
 		return FALSE
 	. = list()
-	var/turf/T = get_turf(C)
-	C.add_splatter_floor(T)
-	playsound(get_turf(C), 'sound/misc/splort.ogg', 80, TRUE)
-	for(var/X in C.internal_organs)
-		var/obj/item/organ/O = X
-		var/org_zone = check_zone(O.zone)
+	var/turf/owner_location = get_turf(chest_owner)
+	chest_owner.add_splatter_floor(owner_location)
```

Comment:
might want to instead check for if chest_owner.loc exists and is a turf otherwise you could splatter through containers / maybe runtime in nullspace